.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: "Bug report"
+about: "Report a bug or a crash in Cranelift."
+labels: 'bug'
+
+---
+
+Thanks for opening a bug report on Cranelift! Please answer the questions below
+if they're relevant and delete this text before submitting.
+
+- What are the steps to reproduce the issue? Can you include a CLIF test case,
+  ideally reduced with the `bugpoint` clif-util command?
+- What do you expect to happen? What does actually happen? Does it panic, and
+  if so, with which assertion?
+- Which Cranelift version / commit hash / branch are you using?
+- If relevant, can you include some extra information about your environment?
+  (Rust version, operating system, architecture...)
```

Comment:
I think it might be helpful to provide a `describe-env.sh` script (and link to it here) to collect the information for these last two questions. Maybe the script doesn't work in all environments but this type of thing could help make it quicker to fill out a bug report and provide a standard format for the information.

Reply:
It's a great idea! I'll open a follow-up for this, because I have no idea what this script would look like on OS that are not Linux.